src/project/tests/mocks/msczreadermock.h

```diff
@@ -26,7 +26,7 @@
 
 #include "project/imscmetareader.h"
 
-namespace mu::notation {
+namespace mu::project {
 class MsczReaderMock : public project::IMscMetaReader
```

Comment:
(This class used to live in `src/notation/tests`, but it's only used by files in `src/project/tests`.)